pkg/genevalogging/genevalogging.go

```diff
@@ -110,6 +110,27 @@ const (
 	DB /var/lib/fluent/audit
 	Parser audit
 
+[FILTER]
```

Comment:
checking k8s.io/apiserver/pkg/apis/audit/v1/types.go, I think we should also do impersonatedUser (if it exists).

Reply:
added lift filter for impersonatedUser